pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.findologic</groupId>
   <artifactId>xml-export</artifactId>
   <packaging>jar</packaging>
-  <version>1.1-SNAPSHOT</version>
+  <version>1.2-SNAPSHOT</version>
```

Comment:
The next release on GitHub will be 1.4.0 - please make this match here and in the downstream projects.